jsonnet/rules.libsonnet

```
@@ -72,6 +72,10 @@ function(params) {
           expr: 'sum(rate(container_cpu_usage_seconds_total{container="",pod!=""}[5m])) / sum(machine_cpu_cores)',
           record: 'cluster:container_cpu_usage:ratio',
         },
+        {
+          expr: 'sum(irate(container_network_receive_bytes_total{pod!=""}[5m])) BY (pod, interface) + on(pod, interface) group_left(network_name) pod_network_name_info',
+'         record: 'pod:container_network_receive_bytes_total:sum',
```

to follow https://prometheus.io/docs/practices/rules/
```suggestion
          record: 'pod_interface_network:container_network_receive_bytes:irate5m',
```